requirements.yml

```diff
@@ -0,0 +1,5 @@
+- src: geerlingguy.filebeat
+- src: geerlingguy.java
+- src: https://github.com/jesusaurus/ansible-role-logstash
+  version: arbitrary-outputs
+  name: geerlingguy.logstash
```

Comment:
So i'm not sure how i feel about the galaxy requirements. I trust the geerlingguy one's more than most but i'd almost prefer we either vendored them or just wrote our own versions. 

This is a bigger conversation.